Source/Lib/Codec/EbDefinitions.h

```diff
@@ -2049,7 +2049,7 @@ static const EB_U8 SearchAreaHeightOq[5][MAX_SUPPORTED_MODES] = {
 	{  64,   64,   64,   64,   16,   13,   13,    9,    9,    7,    7,    7,    7 },
 	{  64,   64,   64,   64,   16,    9,    9,    7,    7,    7,    7,    7,    7 },
 	{  64,   64,   64,   64,   16,   13,   13,    9,    9,    7,    7,    7,    7 },
-	{  64,   64,   64,   64,    9,    9,    9,    9,    7,    7,    7,    7,    7 }
+	{  64,   64,   64,   64,    9,    9,    9,    9,    7,    7,    7,    5,    5 }
 };
```

Comment:
{  64,   64,   64,   64,   24,   16,   13,    9,    9,    7,    7,    7,    5 } is the SQ config.
Is this a typo?